website/package.json

```
@@ -14,11 +14,11 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-beta.3",
-    "@docusaurus/plugin-client-redirects": "^2.0.0-beta.3",
-    "@docusaurus/plugin-sitemap": "^2.0.0-beta.3",
-    "@docusaurus/preset-classic": "^2.0.0-beta.3",
-    "@docusaurus/theme-search-algolia": "^2.0.0-beta.3",
+    "@docusaurus/core": "2.0.0-beta.14",
+    "@docusaurus/plugin-client-redirects": "2.0.0-beta.14",
+    "@docusaurus/plugin-sitemap": "^2.0.0-beta.14",
```

shall we freeze the versions at `2.0.0-beta.14` ?
```suggestion
    "@docusaurus/plugin-sitemap": "2.0.0-beta.14",
```

---

2.0.0-beta.15 is the latest stable version, did you test the beta-15? If that works, can you please freeze it to beta-15?

---

It's not a good practice to freeze the versions in the package.json, the versions will be frozen in package-lock.json in our local, but I see your point, if you want stability we can freeze the version but, we should also once in a while try to upgrade to the latest stable version which might have security and other critical bug fixes.

---

Good call.  So I triaged that the issue is actually due to the latest version of docusaurus, `2.0.0-beta.15` released yesterday.  Freezing it to `2.0.0-beta.14` solves the issue.

@vingov do you know why docusaurus has `beta` in its versions?  Are they still experimental?  For now, sticking to one version saves us time from debugging such issues again in near future.